yt_dlp/extractor/lumni.py

```diff
@@ -0,0 +1,23 @@
+from .common import InfoExtractor
+from .francetv import FranceTVIE
+
+
+class LumniIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?lumni\.fr/video/(?P<id>\w+)'
```

Comment:
Right now you are only matching letters or numbers for the 'id', it would be better if you matched letters, numbers and dashes like this:

```python
    _VALID_URL = r'https?://(?:www\.)?lumni\.fr/video/(?P<id>[\w-]+)'
```

This way, the display id for 'https://www.lumni.fr/video/l-homme-et-son-environnement-dans-la-revolution-industrielle' would be 'l-homme-et-son-environnement-dans-la-revolution-industrielle' instead of just 'l'